src/config/xorg/index.md

```diff
@@ -51,3 +51,35 @@ Void also provides proprietary video drivers, which are available in the
 | NVIDIA     | Proprietary | 500+                            | `nvidia`       |
 | NVIDIA     | Proprietary | 300/400 Series                  | `nvidia390`    |
 | NVIDIA     | Proprietary | GeForce8/9 + 100/200/300 Series | `nvidia340`    |
+
+## Input Drivers
+
+A number of input drivers are available for Xorg. If `xorg-minimal` was
+installed and a device is not responding, or behaving unexpectedly, a different
+driver may correct the issue. These drivers can grab everything from power
+buttons to mice and keyboards.
+
+| Driver                |
+|-----------------------|
+| `xf86-input-evdev`    |
+| `xf86-input-joystick` |
+| `xf86-input-libinput` |
+| `xf86-input-mtrack`   |
+| `xf86-input-synaptics`|
+| `xf86-input-vmmouse`  |
+| `xf86-input-wacom`    |
+
+## Xorg config
+
+An Xorg config for a keyboard may look something like a file
+`/etc/X11/xorg.conf.d/30-keyboard.conf` with the contents.
```

Comment:
Please emphasize that this is almost never necessary.

Reply:
added that the config is to use a specific driver

Reply:
Everything else looks great, but I would appreciate a stronger notice that this is almost never needed.

"Although it is almost never necessary, an Xorg config for ..."